src/services/QuoteSaveService.ts

```
@@ -0,0 +1,95 @@
+    };
+    
+    // Save to Firestore
+    const quotesCollection = collection(db, 'quotes');
+    const docRef = await addDoc(quotesCollection, quoteToSave);
+    
+    return {
+      success: true,
+      quoteId: docRef.id
+    };
+    
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)